### PR TITLE
[Feature/AES-1546-doubleimage-link]: Double Image component

### PR DESCRIPTION
**Pull request by kevin-ho87**
#### Tickets
[AES-1546](https://aesoponline.atlassian.net/browse/AES-1546)
[AES-1547](https://aesoponline.atlassian.net/browse/AES-1547)
[AES-1500](https://aesoponline.atlassian.net/browse/AES-1500)

#### Description
This PR combines 3 tickets - updating the Double Image component. Adding/Updating:
Titles, image/video assets, captions as rich text, links to wrap image and inside title.
image

### CODE REVIEW

**Comment by beau-charman** [comment]
Have a look at those comments, and give this branch a rebase. 🙌

**Comment by kevin-ho87** [comment]
> Have a look at those comments, and give this branch a rebase. 🙌

hey @beau-charman , what comments are you referring to?

**Comment by beau-charman** [comment]
The comments on the code in this PR 

Screen Shot 2020-07-29 at 12 17 09 pm

**Comment by beau-charman** [comment]
Are the comments showing up? I think I had to click 'request changes'

**Comment by kevin-ho87** [comment]
> Are the comments showing up? I think I had to click 'request changes'

Yes they are now. Thanks!

**Comment by kevin-ho87** [comment]
@beau-charman I have made updates.

File: src/components/DoubleMedia/DoubleMedia.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import cx from 'classnames';
+import MediaBlock from './components/MediaBlock';
+import styles from './DoubleMedia.module.css';
+
+const DoubleMedia = ({ mediaOne, mediaTwo, className, theme }) => {
+  const classSet = cx(styles.base, styles[theme], className);
+
+  return (
+    <div className={classSet}>
+      {mediaOne && <MediaBlock {...mediaOne} theme={theme} />}
```
Comment (beau-charman) [inline src/components/DoubleMedia/DoubleMedia.js:12]: Yes! this is great. so clean, love it.

Comment (kevin-ho87) [reply src/components/DoubleMedia/DoubleMedia.js:12]: 🙏

**Pull request merged**